src/styles/blocks/star-block.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  padding: 0;
+  margin: auto 0;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--1 .stars__star:nth-child(-n + 1),
+  &--2 .stars__star:nth-child(-n + 2),
+  &--3 .stars__star:nth-child(-n + 3),
+  &--4 .stars__star:nth-child(-n + 4),
+  &--5 .stars__star:nth-child(-n + 5) {
```

Comment:
The BEM naming convention is not correctly followed here. The modifier class '&--1', '&--2', etc., should be applied to the block element directly, not as a separate class. Consider using '.stars--1', '.stars--2', etc., to maintain consistency with BEM conventions.